configs/pvt/retinanet_pvt_l_fpn_1x_coco.py

```diff
@@ -0,0 +1,17 @@
+_base_ = [
+    'retinanet_pvt_t_fpn_1x_coco.py',
+]
+model = dict(
+    pretrained='https://github.com/whai362/PVT/'
```

Comment:
We use init_cfg to do the initialization now
you can refer to 
https://github.com/open-mmlab/mmdetection/blob/604bfe9618533949c74002a4e54f972e57ad0a7a/configs/atss/atss_r50_fpn_1x_coco.py#L16